cmd/clusterctl/client/repository/components.go

```diff
@@ -189,7 +189,7 @@ func NewComponents(input ComponentsInput) (Components, error) {
 	}
 
 	processedYaml := input.RawYaml
-	if !input.Options.SkipVariables {
+	if !input.Options.SkipVariableProcess {
```

Comment:
Should this be SkipComponentsTemplateProcess (or simply SkipTemplateProcess)?
Also, should we return here when skipping template process, given that all the following code assumes we have a valid yaml, which it is not the case for RawYaml